templates/sb-development.ini.erb

```diff
@@ -3,13 +3,16 @@
 
 # defaults to 'False'
 debug = <%= @sideboard_debug_enabled %>
-priority_plugins = uber,
+priority_plugins = uber, panels
 
 [cherrypy]
 server.socket_host = <%= @socket_host %>
 server.socket_port = <%= @socket_port %>
 tools.sessions.timeout = 4320  # 30 days (in minutes)
 
+server.thread_pool = 100
+server.socket_timeout = 1
```

Comment:
We previously had 10 threads with a 10 second timeout.  Seems bad.  Now we have 100 threads with a 1 second timeout.  Seems good.